src/app/+my-dspace-page/my-dspace-results/my-dspace-results.component.ts

```diff
@@ -37,6 +37,7 @@ export class MyDSpaceResultsComponent {
    */
   @Input() viewMode: ViewMode;
 
+  context = Context.Submission;
```

Comment:
I think that context here can't be fixed, but should change on mydspace configuration used. So I image two different context for the moment `workspace` and `workflow`.

it may be necessary if you want render the same object model different from the current configuration.
For example if we'd show all submission rejected by a controller in the `All tasks` section, we'd need to have two context to render `workspaceitem` differently.

before this PR it was enough to just do [that](https://github.com/4Science/dspace-angular/blob/hasselt_dev_angular6/src/app/shared/object-collection/shared/workspaceitem-rejected-my-dspace-result.model.ts)

Reply:
@atarix83 Thanks for the feedback.
I tried out what happens on the demo when one rejects a workflow item. It disappears from the 'All tasks' section on the MyDSpace page (and becomes a WSI again). This also was the behaviour in DSpace 6 and earlier.

Is there a use case that's applicable to our current code base that would break by using a single context?
It's very simple to add a new context if need be, and for now leaving it out would keep everything a bit cleaner.

Reply:
When a workflowitem is rejected becames a worspaceitem again for the submitter. The example I made, that could be considered an applicable use case, is to allow the controller to keep track of the workflowitems rejected that appears in the 'All tasks' as workspaceitems.

It's true that now it's not used but is also true that before this PR this it was possible to make this distinction simply passing the name of the configuration to the `searchResultFor` decorator

Reply:
That's true, you already implemented it so maybe it's not a good idea to remove the option. I made sure the context is now chosen based on the current configuration.